R/navbar.R

```diff
@@ -11,13 +11,15 @@ data_navbar <- function(pkg = ".", depth = 0L) {
   components[names(components_meta)] <- components_meta
   components <- purrr::compact(components)
 
-  # Any unplaced components go to the right of the left navbar
   right_comp <- intersect(structure$right, names(components))
   left_comp <- intersect(structure$left, names(components))
-  extra_comp <- setdiff(names(components), c(left_comp, right_comp))
+
+  # Warn about unplaced components
+  check_missing_nav_links(components, c(left_comp, right_comp), pkg)
```

Comment:
I think you can omit this warning? Otherwise you'll always get a warning when removing a base component.